metadetect/lsst_metadetect.py

```diff
@@ -88,13 +91,21 @@ def run_metadetect(
             assert len(mbobs) == 1, 'no multiband for now'
             assert len(mbobs[0]) == 1, 'no multiepoch'
             obs = mbobs[0][0]
+
             exposure = obs.exposure
+
+            if config['deblend']:
+                noise_image = obs.meta['extra_noise_exp'].image
```

Comment:
This is definitely monkey patching to some extent. We could add another property to the ngmix obs.

Reply:
If you prefer we could create a list of lists of these and use them as needed.

Reply:
But that has the issue of keeping those lists in sync.

The draw back of a new obs property is scope creep or feature creep, however you want to call it

Reply:
Right. My question is why can't we generate this noise image internally?

Reply:
We are using the same noise image for all metacal images (see comments below and link)

Thus we would be regenerating the same noise field five times

Reply:
Ack NVM. You are making it internally. ignore this.